projects/cdk/directives/active-zone/tests/active-zone.directive.spec.ts

```diff
@@ -0,0 +1,154 @@
+import {CommonModule} from '@angular/common';
+import {Component} from '@angular/core';
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+import {FormControl, ReactiveFormsModule} from '@angular/forms';
+import {TuiActiveZoneModule} from '@taiga-ui/cdk';
+import {configureTestSuite} from '@taiga-ui/testing';
+
+describe(`TuiActiveZoneDirective`, () => {
```

Comment:
Check out this PR:
https://github.com/Tinkoff/taiga-ui/pull/4420
I wonder if jest can properly handle that :)